src/lib/viewers/iframe/__tests__/IFrameLoader-test.js

```diff
@@ -43,6 +51,7 @@ describe('lib/viewers/iframe/IFrameLoader', () => {
             };
             const viewer = IFrameLoader.determineViewer(iframe.options.file, [], viewerOptions);
             expect(viewer).toEqual(viewerInstance);
+            expect(IFrameLoader.getViewers().some(v => v.EXT.includes('boxdicom'))).toEqual(!disableDicom);
```

Comment:
Do we need this assertion if we already have the one above it?

Reply:
We could, the assertion checks if the list of viewers is correct based on the disableDicom viewer option. While the above assertion checks if the viewer instance is correct.